k8s.go

```diff
@@ -0,0 +1,247 @@
+package profitbricks
+
+import "net/http"
+
+type KubernetesClusters struct {
+	// URL to the collection representation (absolute path)
+	// Read Only: true
+	// Format: uri
+	Href string `json:"href,omitempty"`
+
+	// Unique representation for Kubernetes Cluster as a collection on a resource.
+	// Read Only: true
+	ID string `json:"id,omitempty"`
+
+	// Array of items in that collection
+	// Read Only: true
+	Items []*KubernetesCluster `json:"items"`
```

Comment:
- I haven't understood yet when you use pointers and when you don't. Can you explain?
- Same as for a cluster's nodepool entities: The things in `Items` are not full-fledged clusters.

Reply:
- swagger generate decided to do it like this. But I'll change it to match the other models.
- They are, if the depth is high enough